documentation/mkdocs-base.yml

```diff
@@ -47,8 +47,8 @@ theme:
     - content.tabs.link
     - content.code.copy
     - navigation.tracking
-    - content.action.edit
-#    - content.action.view
+#    - content.action.edit
+    - content.action.view
```

Comment:
Is it an "view" action now? I tested the preview and it seems like the user won't be taken to the edit page.